TransferList/ButtonList.js

```diff
@@ -80,6 +92,14 @@ ButtonList.propTypes = {
 	 */
 	disabled: PropTypes.bool,
 
+	/**
+	 * Disables Remove Button when items are not selected or dragged
+	 *
+	 * @type {Boolean}
+	 * @private
+	 */
+	elementsAreDragged: PropTypes.bool,
```

Comment:
if this prop only applies to `Remove button` we need to find a better name. maybe `trashBinActive`? any suggetions are welcomed